test/test_cuda.py

```diff
@@ -2116,12 +2116,12 @@ def test_sum_cpu_gpu_mismatch(self):
         x = torch.randn(20, dtype=torch.float32, device='cuda')
         y = torch.randn(1, dtype=torch.float32)
         with self.assertRaisesRegex(RuntimeError,
-                                    'expected backend CPU and dtype Float but got backend CUDA and dtype Float'):
+                                    'expected device cpu and dtype Float but got device cuda:0 and dtype Float'):
```

Comment:
are we actually guaranteed to run on cuda:0 here?  I forget if we enforce that the default device is set to 0 between tests.

Reply:
I think so (all the tests pass). Would you prefer the definition of x to say 'cuda:0' instead of 'cuda'?

Reply:
sure, that sounds good.